apps/posts/src/views/posts/Posts.tsx

```
@@ -0,0 +1,20 @@
+import {Button, H1, Page} from '@tryghost/shade';
+import {useNavigate} from '@tryghost/admin-x-framework';
+
+interface postAnalyticsProps {};
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)